### PR TITLE
Add Arthera testnet network support

### LINKED ISSUE #863: Add Arthera testnet network support
### Expected behavior

Being able to use Arthera testnet in the supported networks.

### Actual behavior

Arthera testnet is not a supported network yet.

### Steps to reproduce the behavior

Browse https://snapshot.org/#/?filter=networks&q=Arthera

### PR DESCRIPTION
**Pull request by bertux**
Fixes #863  .

Changes proposed in this pull request:
- Add Arthera network logo
- Add Arthera testnet network RPC details
- Add Arthera testnet network contract address

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1488,6 +1488,17 @@
...
   },
+  "10243": {
+    "key": "10243",
+    "name": "Arthera Testnet",
+    "shortName": "Arthera_",
+    "chainId": 10243,
+    "network": "testnet",
+    "multicall": "0x27c7FC597aD2E81C4c1cA1769972f79DaF042Da7",
+    "rpc": ["https://rpc-test.arthera.net"],
+    "explorer": "https://explorer-test.arthera.net",
```
Comment (ChaituVR) [inline src/networks.json:1499]:
```suggestion
    "explorer": {
      "url": "https://explorer-test.arthera.net"
    },
```

File: src/networks.json [hunk 2]
```diff
@@ -1488,6 +1488,19 @@
...
+    "name": "Arthera Testnet",
+    "shortName": "Arthera_",
+    "chainId": 10243,
+    "network": "testnet",
+    "multicall": "0x27c7FC597aD2E81C4c1cA1769972f79DaF042Da7",
+    "rpc": ["https://rpc-test.arthera.net"],
+    "explorer": {
+      "url": "https://explorer-test.arthera.net"
+    },
+    "logo": "ipfs://QmYQp3e52KjkT4bYdAvB6ACEEpXs2D8DozsDitaADRY2Ak"
```
Comment (ChaituVR) [inline src/networks.json:1502]:
```suggestion
    "start": 10523,
    "logo": "ipfs://QmYQp3e52KjkT4bYdAvB6ACEEpXs2D8DozsDitaADRY2Ak"
```

File: src/networks.json [hunk 3]
```diff
@@ -1488,6 +1488,20 @@
     },
     "logo": "ipfs://QmWG1p7om4hZ4Yi4uQvDpxg4si7qVYhtppGbcDGrhVFvMd"
   },
+  "10243": {
+    "key": "10243",
+    "name": "Arthera Testnet",
+    "shortName": "Arthera_",
+    "chainId": 10243,
+    "network": "testnet",
+    "multicall": "0x27c7FC597aD2E81C4c1cA1769972f79DaF042Da7",
```
Comment (ChaituVR) [inline src/networks.json:1497]:
```suggestion
    "testnet": true,
    "multicall": "0x27c7FC597aD2E81C4c1cA1769972f79DaF042Da7",
```

**Review by ChaituVR — APPROVED** [review]

**Comment by lperello** [comment]
Thank you @ChaituVR 
I will not forget you help
Laurent

**Pull request merged**